app/api/schema/settings.py

```diff
@@ -164,6 +164,8 @@ class Meta:
 
     # Stripe secret key
     stripe_secret_key = fields.Str(allow_none=True)
+    stripe_client_id = fields.Str(allow_none=True)
+    stripe_publishable_key = fields.Str(allow_none=True)
```

Comment:
@mrsaicharan1 Why this change, arent this already there?

Reply:
@kushthedude This wasn't required. I will change this.